src/app/webbrowser/Browser.qml

```diff
@@ -844,16 +844,16 @@ Common.BrowserView {
             target: browser.currentWebview
             onLoadingChanged: {
                 if (browser.currentWebview.loading) {
-                    chrome.state = "shown"
+                    chrome.state = "shown";
                 } else if (browser.currentWebview.isFullScreen) {
-                    chrome.state = "hidden"
+                    chrome.state = "hidden";
```

Comment:
These changes, while useful, are not necessary for this PR. It'd be best if they could be removed from the commit they are added, or split out into their own commit so they're at least atomic.

Reply:
luckily it was like the example in https://stackoverflow.com/a/9103397 : )
So I've removed the changes from the loadingChanged function.